discopop_explorer/PETGraphX.py

```diff
@@ -173,53 +193,46 @@ class PETGraphX(object):
     g: nx.MultiDiGraph
     reduction_vars: List[Dict[str, str]]
     main: CUNode
-    pos: Dict
 
-    def __init__(self, g: nx.MultiDiGraph, reduction_vars: List[Dict[str, str]], pos):
-        self.g = g
+    def __init__(self, cu_dict: Dict[str, ObjectifiedElement], dependencies_list: List[DependenceItem],
```

Comment:
Here (and the following lines) it seems to revert the change from b9102d891e7a05805232f536fa16e1ad11396d67. Why?

Reply:
You´re right. Sorry. I´ll revert This change.

Reply:
Should be fixed now.